src/Extension/AssetTrait.php

```diff
@@ -180,16 +180,16 @@ private function normalizeAsset(FileAssetInterface $asset)
             return;
         }
 
-        $file = $this->getWebDirectory()->getFile($asset->getPath());
+        $app = $this->getContainer();
+
+        $file = $app['filesystem']->getFile("extensions_assets://{$this->getId()}/$path");
```

Comment:
Yeah I found that as I was finishing up yesterday. Extension assets are copied to this directory by their composer id, right?